scripts/docker/ubuntu/focal/Dockerfile

```diff
@@ -0,0 +1,106 @@
+FROM ubuntu:focal
```

Comment:
Thank you, this looks great! We should also add it to the normal pipeline.

Reply:
I will do this as soon as possible.